### ISSUE TITLE
SharePoint Server support for Kerberos 

### ISSUE THREAD

**Issue by serenachou** [elastic/connectors#592]
Requested in https://elastic.slack.com/archives/C7LLL50CA/p1679404820729719 

> The connector uses NTLM auth and they are standardized on kerberos. Is there a known workaround or, if not, something on the roadmap to add kerberos support?

Additional requests and upvotes for this functionality in: https://github.com/elastic/enterprise-search-sharepoint-server-connector/issues/69

**Comment by parth-crest** [comment]
Yes, we will look into it and will get back to you with the updates.

**Comment by akanshi-elastic** [comment]
Working on kerberos  to fix OS dependency issue by creating shell script suggest by artem

**Comment by moxarth-crest** [comment]
@danajuratoni can you please close this issue since no updates for a very long time?   https://github.com/elastic/connectors/pull/1141#issuecomment-2101224563

**Comment by artem-shelkovnikov** [comment]
Won't do for now
